src/Commands/CommentAddPullRequestCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Terminus Plugin that contain a collection of commands useful during
+ * the build step on a [Pantheon](https://www.pantheon.io) site that uses
+ * a GitHub PR workflow.
+ *
+ * See README.md for usage information.
+ */
+
+namespace Pantheon\TerminusBuildTools\Commands;
+
+use Pantheon\Terminus\Exceptions\TerminusException;
+
+/**
+ * Pull Request Comment Command
+ */
+class CommentAddPullRequestCommand extends BuildToolsBase
+{
+
+    /**
+     * Add a comment to a specified PR on the repository.
+     *
+     * @authorize
+     *
+     * @command build:comment:add:pr
+     */
+    public function commentAddPullRequest(
+        $options = [
+            'pr_id' => '',
+            'message' => '',
+            'site_url' => ''
+        ])
+    {
+        // Get current repository and commit
+        $remoteUrlFromGit = exec('git config --get remote.origin.url');
+        $prId = $options['pr_id'];
+        if (empty($prId)) {
+            throw new TerminusException( '--pr_id=<id> is required.' );
+        }
+
+        // Create a Git repository service provider appropriate to the URL
+        $this->inferGitProviderFromUrl($remoteUrlFromGit);
+
+        // Ensure that credentials for the Git provider are available
+        $this->providerManager()->validateCredentials();
+
+        // Compile message
+        if (!empty($options['site_url'])) {
+            $message = "[![Visit Site](https://raw.githubusercontent.com/pantheon-systems/ci-drops-8/0.1.0/data/img/visit-site-36.png)](".$options['site_url'].")\n\n".$options['message'];
```

Comment:
We should move the `.png` image into the Build Tools repo, rather than relying on another repository.

Reply:
Do you have a preference as to where I put it? It doesn't look like we have any assets in this repository right now.

Reply:
`assets/images` works

Reply:
Updated in a65beefbcd1d20affdae8e8e1cd555e57b283a40 but also created #281